shared/models/videos/abuse/video-abuse-create.model.ts

```diff
@@ -1,3 +1,6 @@
 export interface VideoAbuseCreate {
   reason: string
+  predefinedReasons?: {[key: string]: boolean} // see VideoAbusePredefinedReasonsIn
```

Comment:
I'm not sure to understand why it's not just an array of `VideoAbusePredefinedReasons` (~ number)?

Reply:
The goal is to make using the API easier.

Reply:
Then use an array of strings if you don't think enum numbers are clear enough